src/Home.tsx

```diff
@@ -134,6 +134,8 @@ const Home = ({
 		} catch (error: any) {
 			if (error.message.indexOf('showSaveFilePicker is not a function') !== -1) {
 				onModal('failed', 'Your current browser does not support this site\'s functionality. For the best experience, please use Chrome 86+ (recommended).', 'about');
+			} else if (error.message.indexOf('The user aborted a request.') !== -1) {
+				onModal('failed', 'The user aborted a request.');
```

Comment:
Aborted by user?

Reply:
if user click on cancel btn in file explorer